### DISCUSSION TITLE
quasar typescript alias path imports work only for interfaces

### DISCUSSION THREAD

**Discussion by JumaVandam** [quasarframework/quasar#14994] [General - Components / Directives / etc] [answered]
Hello, I have a project on quasar version 2.10.0 on typescript version 4.8.4, @quasar/app-webpack version 3.6.2, a project on the backend on nest js and there is a folder called common with data types to use for the frontend and backend. I added my path to tsconfig.json pointing to the common folder and on the frontend if I import enams, classes, variables and use them in the code, then I get the error: 

> Module not found: Can't resolve imported dependency "@shared/index" Did you forget to install it? You can run: npm install --save @shared/index.

 But for some reason there are no problems with import and use the interfaces. 
I can't solve this problem myself please help.

![Снимок](https://user-images.githubusercontent.com/82858978/204204951-a0c35165-a83d-4d93-bd96-4945b855a9cb.PNG)
![фыв](https://user-images.githubusercontent.com/82858978/204205027-87f16a7c-1df8-43ca-9f77-be68f9370b7b.PNG)
![Снимок 3](https://user-images.githubusercontent.com/82858978/204205079-f02b2f30-5b81-49d1-a102-1d37c6611949.PNG)

**Comment by metalsadman** [comment, marked as answer]
Also set your aliases here https://quasar.dev/quasar-cli-webpack/handling-webpack#adding-webpack-aliases

**Reply by JumaVandam** [reply]
Thank you very much man, its works. 😘
Worked with chainWebpack but extendWebpack doesn't.
